packages/core/src/lib/context-menu/context-menu.svelte

```diff
@@ -17,13 +17,21 @@ import cx from 'classnames';
 /** ID attribute of the menu element. */
 export let id: string;
 
+/** Accessible label of the menu. */
+export let label: string | undefined = undefined;
+
+/** ID of the element ID that labels the menu. */
+export let labeledBy: string | undefined = undefined;
```

Comment:
[minor] labelledBy vs labeledBy? might be good to keep consistent with the aria class

Reply:
I cannot spell 🤦

Reply:
have faith in your spelling abilities!